webui/pom.xml

```diff
@@ -13,7 +13,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.alluxio</groupId>
-  <artifactId>alluxio-ui</artifactId>
+  <artifactId>alluxio-webui</artifactId>
```

Comment:
Should we just call this `webui`?

Reply:
our POM artifact convention is to make `foo/` into `alluxio-foo`, like `integration` module:
` <artifactId>alluxio-integration</artifactId>`

Reply:
got it, thanks.